wski18n/i18n_resources.go

```diff
@@ -1,463 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Hmm, why don't we need this file anymore?

Reply:
This is auto-generated from the go-bindata command, that should be run on every build.  As it MUST be auto-generated, doesn't seem to make sense to have it checked in, and be stale.  MIght cause confusion if you have never had to run go-bindata and you update the resource strings and don't see your new messages showing up.